docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,16 @@
 
 ## May 2024
 
+**23 May 2024**
+
+Frontend version: 1.0.0-20240520.1421 | Backend version: 1.105.3-20240521.1554
+
+| Type  | Change| Description |
+|-------|-------|-------------|
+| **Change** | Enable TechPass admin to update user email/username on identities page | Tenants can now expect to receive changes in user's email/upn via our ```user-updated webhook``` event. |
+| **Change** | Support managing client secrets for SAML apps | We haveve reintroduced the ability to add and delete client secrets for SAML apps. This functionality is available via the portal under our automation API across all environments. Tenants can now manage secrets to securely configure SAML apps as required. |
```

Comment:
typo 'haveve'.

'available via the portal under our automation API' sounds wrong, cuz automation API is separated from the portal. The change for this release is to make it available on the portal only. It was already made available for Automation API in previous release.

And instead of 'reintroduce' can just say 'introduced' cuz this will be new to all other tenants besides GCC.

so I think we can just put 'We have introduced the ability to add and delete client secrets for SAML apps on the portal.'